pkg/polarion-generator/test_cases_generator.go

```diff
@@ -70,6 +73,55 @@ func addCustomField(customFields *polarion_xml.TestCaseCustomFields, id, content
 		})
 }
 
+func addLinkedWorkItem(linkedWorkItems *polarion_xml.TestCaseLinkedWorkItems, id string) {
+	linkedWorkItems.LinkedWorkItems = append(
+		linkedWorkItems.LinkedWorkItems, polarion_xml.TestCaseLinkedWorkItem{
+			ID:   id,
+			Role: "verifies",
+		})
+}
+
+func parseTagsFromTitle(testCase *polarion_xml.TestCase, title string, component string, projectID string) {
+	posneg := "positive"
+	caselevel := "component"
+	criticality := "medium"
+	comp := component
```

Comment:
Why not using component directly?

Reply:
you got me :)
what i really wanted to do here, was to set it to component in case the test doesnt have its own component set.
will fix

Reply:
fixed